src/resources/Pipelines/StatementGroups/tests/StatementGroups.spec.ts

```diff
@@ -25,6 +25,16 @@ describe('StatementGroups', () => {
             expect(api.get).toHaveBeenCalledTimes(1);
             expect(api.get).toHaveBeenCalledWith(StatementGroups.getBaseUrl(pipelineId));
         });
+
+        it('should make a GET call with a filter', () => {
+            const pipelineId = '️🍰';
+            const filter = 'nameOfCondition';
+            const expectedUri = StatementGroups.getBaseUrl(pipelineId) + '?filter=' + encodeURIComponent(filter);
```

Comment:
Hardcoded expectation would make for a stronger test.

Reply:
Done